error: variable 'kvzChroma' is used uninitialized

Building w/ Clang 17 fails w/
```
C:/_/B/src/libheif-1.17.6/libheif/plugins/encoder_kvazaar.cc:480:12: error: variable 'kvzChroma' is used uninitialized whenever 'if' condition is false [-Werror,-Wsometimes-uninitialized]
    480 |   else if (chroma == heif_chroma_444) {
        |            ^~~~~~~~~~~~~~~~~~~~~~~~~
  C:/_/B/src/libheif-1.17.6/libheif/plugins/encoder_kvazaar.cc:[544](https://github.com/msys2/MINGW-packages/actions/runs/7408100857/job/20155643547#step:11:545):45: note: uninitialized use occurs here
    544 |   kvz_picture* pic = api->picture_alloc_csp(kvzChroma, encoded_width, encoded_height);
        |                                             ^~~~~~~~~
  C:/_/B/src/libheif-1.17.6/libheif/plugins/encoder_kvazaar.cc:480:8: note: remove the 'if' if its condition is always true
  
    480 |   else if (chroma == heif_chroma_444) {
  
        |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
  C:/_/B/src/libheif-1.17.6/libheif/plugins/encoder_kvazaar.cc:456:3: note: variable 'kvzChroma' is declared here
  
    456 |   kvz_chroma_format kvzChroma;
  
        |   ^
  
  1 error generated.
```

--- a/libheif/plugins/encoder_kvazaar.cc
+++ b/libheif/plugins/encoder_kvazaar.cc
@@ -35,6 +35,7 @@ extern "C" {
 
 static const char* kError_unspecified_error = "Unspecified encoder error";
 static const char* kError_unsupported_bit_depth = "Bit depth not supported by kvazaar";
+static const char* kError_unsupported_chroma = "Unsupported chroma type";
 //static const char* kError_unsupported_image_size = "Images smaller than 16 pixels are not supported";
 
 
@@ -491,6 +492,13 @@ static struct heif_error kvazaar_encode_image(void* encoder_raw, const struct he
     chroma_height_shift = 0;
     input_chroma_width = input_width;
     input_chroma_height = input_height;
+  }
+  else {
+    return heif_error{
+      heif_error_Encoder_plugin_error,
+      heif_suberror_Unsupported_image_type,
+      kError_unsupported_chroma
+    };
   }
 
   if (chroma != heif_chroma_monochrome) {

